corehq/motech/dhis2/repeaters.py

```diff
@@ -349,7 +349,7 @@ def _migration_get_couch_model_class(cls):
         return Dhis2Repeater
 
     def _wrap_schema_attrs(self, couch_object):
-        setattr(couch_object, 'dhis2_config', Dhis2Config.wrap(getattr(self, 'dhis2_config')))
+        setattr(couch_object, 'dhis2_config', Dhis2Config.wrap(self.dhis2_config))
```

Comment:
Similar to @snopoke's comment, I'm not sure why you're using `setattr()` here. Is this the same as
```python
        couch_object.dhis2_config = Dhis2Config.wrap(self.dhis2_config)
```
or does `setattr()` do something, or avoid something, that setting the property directly doesn't do the way you want?

Reply:
There is not any specific reason to use setattr. I guess I used it because it is being used by the migration code. But I guess the point that you are trying to make here is - since we are sure that the property would exist then setting it directly would be more readable?

Reply:
Yes. I think the migration code uses `setattr()` because the attribute name is passed in a variable. But in this situation, we know the attribute name is 'dhis2_config'. So if there is no special reason to use `setattr()`, then I reckon just setting the attribute normally is more readable.